target-app/Home/Views/HomeViewController.swift

```diff
@@ -24,22 +42,40 @@ class HomeViewController: UIViewController {
         setupMapConstraints()
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        LocationManager.shared.requestLocationAuthorization()
+        
+        if CLLocationManager.locationServicesEnabled() {
+            locationManager.delegate = self
+            locationManager.desiredAccuracy = kCLLocationAccuracyBest
+            locationManager.startUpdatingLocation()
+        }
+        mapView.delegate = self
+        mapView.mapType = .standard
+        mapView.isZoomEnabled = true
+        mapView.isScrollEnabled = true
+        
```

Comment:
If you want, you can set all this properties inside the lazy var mapView when youre creating it, that keeps all the mapView configuration in one place

Reply:
done!